src/js/export.js

```diff
@@ -57,19 +58,34 @@
 
   }
 
+  function destroy(modal) {
+    document.body.removeChild(modal);
+  }
```

Comment:
I think the only thing you remove is _the one_ modal, which is now stored in a almost-global variable.
```
function destroy(evt) {
    document.body.removeChild(modal);
}
```

And @lines 36-41:
```
document.getElementById('exportCloseBtn").addEventListener('click', destroy, false)
document.getElementById('exportCancelBtn").addEventListener('click', destroy, false)
```